mysql-test/suite/innodb/t/deadlock_stats.test

```diff
@@ -0,0 +1,10 @@
+let $engine=innodb;
+let $engine_start_txn=start transaction;
+let $prior_engine_lwt=@prior_innodb_lock_wait_timeout;
```

Comment:
I think the "prior" variables aren't strictly needed for correctness. For simplicity, why don't we just use "prior_lock_wait_timeout" as variable name?